### PR TITLE
Add case studies

### LINKED ISSUE #2189: Add some case studies
We've accumulated a number of them, so we should make them available on the website, starting with:

* [x] Learning Machine
* [x] Merdeces-Benz R&D

### PR DESCRIPTION
**Pull request by cnunciato**
Fixes #2189.

### CODE REVIEW

File: content/why-pulumi/delivering-cloud-native-infrastructure-as-code.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ layout: whitepaper
 
 meta_desc: In this paper, we present a consistent cloud programming model using general purpose programming languages for interacting with cloud resources.
 
+exec_summary: To a first approximation, all developers are cloud developers, all applications are cloud native, and all operations are cloud-first. Yet, there is a lack of a consistent approach to delivering cloud native applications and infrastructure. The tools and processes differ by technology generation, and even by cloud vendor, and so deny the full potential of cloud native application delivery.
```
Comment (joeduffy) [inline content/why-pulumi/delivering-cloud-native-infrastructure-as-code.md]: Just to note, I'm hoping we can retire this whitepaper in early 2020. It reflects our positioning from about a year ago -- which wasn't working quite as well as the new infrastructure as code oriented approach.

**Comment by cnunciato** [comment]
Just to reflect status here, this PR's waiting on some content and 👍 from @ericrudder -- we chatted this morning, but in case others are wondering what the status is, it's close and just needs a bit of clueful wordsmithing.

**Comment by cnunciato** [comment]
Moving this out to M31.

**Comment by joeduffy** [comment]
@cnunciato Definitely eager to get these up. Where will we link to them?

**Comment by cnunciato** [comment]
@joeduffy I've asked @isaac-pulumi to help flesh out the content. I was thinking maybe Why Pulumi might be good first home for them -- what do you think?

**Comment by joeduffy** [comment]
Can we add the customer logos prominently to these pages?

File: content/case-studies/learning-machine/_index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+title: Learning Machine
+type: page
+layout: case-study
+
+meta_desc: |
+    Learn how Learning Machine used Pulumi and TypeScript to streamline its devops processes
```
Comment (joeduffy) [inline content/case-studies/learning-machine/_index.md]:
```suggestion
    Learn how Learning Machine used Pulumi and TypeScript to streamline its DevOps processes
```

File: content/case-studies/mercedes-benz-rdna/_index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+title: Mercedes-Benz Research and Development
+type: page
+layout: case-study
+
+meta_desc: |
+    Learn more about how Mercedes-Benz Research and Development used Pulumi to bring their
+    application and infrastructure teams closer together.
+
+exec_summary: |
+    We need one.
```
Comment (joeduffy) [inline content/case-studies/mercedes-benz-rdna/_index.md]:
```suggestion
Mercedes-Benz Research and Development North America (MBRDNA) chose Pulumi to help them fully leverage modern cloud capabilities, enabling their organization to push the frontiers of innovative automatic research and development. By using technologies like multi-cloud and Kubernetes, and Pulumi's approach to infrastructure as code, their developers and infrastructure engineers are able to work closely to work more productively, and deliver results faster, than ever before.
```

**Pull request merged**